tests/unit/state_manager/test_state_manager.py

```diff
@@ -271,6 +298,8 @@ def test_clean_part(self, new_dir):
         for step in list(Step):
             assert sm._state_db.get(part_name="p1", step=step) is None
 
+        Features.reset()
```

Comment:
same comment here re: `Features.reset()`  not being called on test failure

Reply:
Changed as suggested.